### PR TITLE
PA-12 grid 

### PR DESCRIPTION
**Pull request by hester-g**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -53,8 +53,112 @@
...
     <string name="shelter_dummy_email_hint">enter email here</string>
 
-    <!-- Strings related to login -->
+    <!-- Strings related to Pet Grid -->
+    <string name="pet_grid_picture">This is a pet</string>
+    <string name="pet_grid_title">Pet title</string>
+
+    <!-- Strings related to Settings -->
     <string name="title_activity_login">Sign in</string>
+
```
Comment (novotnej) [inline app/src/main/res/values/strings.xml:59]: This looks like a leftover of some conflict resolution, these **new** strings apart from the two you added are no longer in use

**Review by novotnej — CHANGES_REQUESTED** [review]
- Set the images for pets in DBHelper, currently all pets show up with the default snape
- Remove unused strings in strings.xml
- Missing onclick action when a pet image is clicked

**Comment by hester-g** [comment]
@novotnej 

- when you say set images, do you mean the images for the dogs? also the default should be dumbledore.
- will do
- what should happen when a pet is clicked?

**Comment by novotnej** [comment]
Onclick used to be in the createDummyPetsListView method. Joshua changed it a little bit, so please use the version in PA-7

I don't see the image value being set anywhere, so I presume you may have changed it directly in the database? This is what I see right now

screen

**Comment by hester-g** [comment]
Oh yeah I changed it in the database, is that not what I should do?

**Comment by novotnej** [comment]
It would be nice for us and for the demo if we could all see different
images :D

**Comment by hester-g** [comment]
oh i thought what I had done did that 😬

**Comment by novotnej** [comment]
Nope, the database is generated with each install of the app from the code in dbhelper

**Comment by hester-g** [comment]
wait what i changed was in dbhelper...

**Comment by hester-g** [comment]
Okay @novotnej I think I've done everything

**Comment by hester-g** [comment]
(sorry I accidentally forgot to write actual commit messages for those last three commits so they still all say merge conflict resolution or smth)

**Comment by novotnej** [comment]
The app's not building now, it's the strings
while I think some of those preferences strings are supposed to be there, they definitely shouldn't be coming from your commit since Artem put them there, so it will be some merge gone a bit sideways

in any case you removed two that you actually need for your activity to work.. :D in AS you can use Refactor->remove unused resources (from within the strings.xml file). Please verify the app still works afterwards

**Comment by hester-g** [comment]
Okay @novotnej now I think everythings good! Sorry for the delay i fell asleep waiting for the build to finish 😂

**Comment by novotnej** [comment]
Dude c'mon :D You deleted the dummy images with it and left DummyPetAdapter in there and it wouldn't build with it.. 

I fixed it now, dw

**Review by novotnej — APPROVED** [review]

**Pull request merged**